pyabc/storage/history.py

```diff
@@ -123,64 +143,64 @@ def all_runs(self):
     @with_session
     def _pre_calculate_id(self):
         abcs = self._session.query(ABCSMC).all()
-        if len(abcs) == 1:
-            return abcs[0].id
+        if len(abcs) > 0:
```

Comment:
why this change?

Reply:
what stood there beforehand means that only if there is just 1 analysis in the db, then the id is set to it.
now the id is always set to the lastest added analysis, cause that's what I would expect (it is the most relevant one).